src/QsCompiler/Core/SymbolTable/Namespace.fs

```diff
@@ -3,6 +3,8 @@
 
 namespace Microsoft.Quantum.QsCompiler.SymbolManagement
 
+#nowarn "44" // AccessModifier is deprecated.
```

Comment:
Why is this using the deprecated AccessModifier, and what will be needed to update it?

Reply:
It's only used by `IsDeclarationAccessible`, which is itself deprecated and not used anywhere (but is public). We can just remove `IsDeclarationAccessible` when `AccessModifier` is removed.

Reply:
Ah, I see, thanks for the explanation.